signalr/hubs/samples/7.x/SignalRHubsSample/Snippets/Hubs/ChatHubClientResultsContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace SignalRHubsSample.Hubs;
+// <snippet_HubClientReturnContext>
+public class ChatHubClientResultsContext : Hub
+{
+    async Task SomeMethod(IHubContext<ChatHubClientResultsContext> context)
```

Comment:
This is an odd way to use `IHubContext`. It's probably better from a controller or minimal api:
https://learn.microsoft.com/aspnet/core/signalr/hubcontext?view=aspnetcore-7.0#inject-an-instance-of-ihubcontext-in-a-controller